source/code/include/playfab/PlayFabPlatformUtils.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <chrono>
+#include <ctime>
+#include <iomanip>
+#include <sstream>
+
+namespace PlayFab
+{
+    typedef std::chrono::time_point<std::chrono::system_clock> TimePoint;
+
+    inline TimePoint GetPlayFabTimePointNow()
+    {
+        return std::chrono::system_clock::now();
+    }
+
+    inline time_t GetPlayFabTimeTNow()
+    {
+        time_t now = std::chrono::system_clock::to_time_t(GetPlayFabTimePointNow());
+#if defined(PLAYFAB_PLATFORM_IOS) || defined(PLAYFAB_PLATFORM_ANDROID) || defined(PLAYFAB_PLATFORM_LINUX) || defined(PLAYFAB_PLATFORM_SWITCH)
+        return static_cast<Json::Int64>(now);
+#else // PLAYFAB_PLATFORM_IOS || PLAYFAB_PLATFORM_ANDROID || PLAYFAB_PLATFORM_LINUX || PLAYFAB_PLATFORM_SWITCH
+        return now;
+#endif // PLAYFAB_PLATFORM_IOS || PLAYFAB_PLATFORM_ANDROID || PLAYFAB_PLATFORM_LINUX || PLAYFAB_PLATFORM_SWITCH
+    }
+
+    inline void AppendIntToString(int value, std::string& output)
+    {
+        // itoa is not available in android
+        char buffer[16];
+#if defined(PLAYFAB_PLATFORM_IOS) || defined(PLAYFAB_PLATFORM_ANDROID) || defined(PLAYFAB_PLATFORM_LINUX) || defined(PLAYFAB_PLATFORM_PLAYSTATION) || defined(PLAYFAB_PLATFORM_SWITCH)
+        sprintf(buffer, "%d", value);
+#else // PLAYFAB_PLATFORM_IOS || PLAYFAB_PLATFORM_ANDROID || PLAYFAB_PLATFORM_LINUX || PLAYFAB_PLATFORM_PLAYSTATION || PLAYFAB_PLATFORM_SWITCH
+        sprintf_s(buffer, "%d", value);
+#endif // PLAYFAB_PLATFORM_IOS || PLAYFAB_PLATFORM_ANDROID || PLAYFAB_PLATFORM_LINUX || PLAYFAB_PLATFORM_PLAYSTATION || PLAYFAB_PLATFORM_SWITCH
+        output.append(buffer);
+    }
+
+    inline std::string LocalTimeTToUtcString(time_t now)
```

Comment:
>LocalTime [](start = 23, length = 9)

you use the term "LocalTime" to describe time_t in a number of places. time_t does not represent a local time. it's just an opaque "time" value that usually represents number of seconds since the unix epoch which would not be a locally defined value. Recommend dropping the "local" where appropriate and renaming accordingly. e.g. here I think a better name would just be `TimeAsString` or `TimeToUtcString` or something

Reply:
This is probably a design problem that we have to resolve.
In my past experiments, parsing and printing the output of time_t required shifting/correcting for the timezone, given the input string. If you didn't, the result time_t didn't represent "now" correctly.
There's a unit test that verifies that when we retrieve the server GetTime, it is correctly parsed into a time_t which matches the local clock.
I don't care much about the names of these methods, but I do want to make sure that our interface works.
Once we've verified the interface is working as expected on all platforms, I'll clean up the function names and look for unused code.